### DISCUSSION TITLE
HELP! MAKO ERROR

### DISCUSSION THREAD

**Discussion by tmac31** [RhetTbull/osxphotos#1159] [Q&A]
ModuleNotFoundError: No module named 'mako'

I recently found this repository and followed the documentation step by step. Unfortunately, I kept running into the error seen above and I was wondering if anyone else ran into the same issues or if anyone has the solution to this issue. Any help is appreciated and if any more information is needed please let me know. 

Thank you in advance for the help !

**Comment by RhetTbull** [comment]
What platform (OS) are you using? 
How did you install osxphotos?
Which version of python are you using?

**Reply by RhetTbull** [reply]
Hi @tmac31 any luck resolving this issue?
